packages/storefront-automation/tests/home_test.js

```diff
@@ -0,0 +1,13 @@
+Feature('home');
+
+Scenario('Load a product starting from the home page', I => {
+    I.amOnPage('https://sfcc-team-sample-app.herokuapp.com/');
```

Comment:
Can this url be configurable? If I am running the tests against my branch, it will fail?